hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/dispatcher/MonoTransactionDispatcher.java

```diff
@@ -308,4 +309,10 @@ private void finishTokenFeeScheduleUpdate(@NonNull final HandleContext handleCon
         final var tokenStore = handleContext.writableStore(WritableTokenStore.class);
         requireNonNull(tokenStore).commit();
     }
+
+    private void dispatchNetworkUncheckedSubmit(@NonNull final HandleContext handleContext) {
+        requireNonNull(handleContext);
+        final var handler = handlers.networkUncheckedSubmitHandler();
+        handler.handle(handleContext);
+    }
```

Comment:
Why is this implemented? It will never be used, will it?

Reply:
Correct. PreHandle will always throw an exception so handle will never be called. I went ahead and deleted this.